tests/app/test_queue.py

```diff
@@ -75,33 +74,33 @@ def delete_all_inflight(self, redis):
             redis.delete(key)
 
     @pytest.mark.serial
-    def test_put_mesages(self, redis, redis_queue):
+    def test_put_mesages(self, app, redis, redis_queue):
```

Comment:
This is not necessary to define the `app` mock as a parameter because of the `autouse=True` usage [at this line](https://github.com/cds-snc/notification-api/pull/1451/files#diff-c9d36498371981109d1bf2de7b12bb84964b6f4a6b55a03af7bf94ca02781366R22).

Reply:
I am sure i understand you here.. `app` has been added so we have access to the app context and config.. so we can `app.config["NOTIFY_ENVIRONMENT"]) `